init-mysql.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -x
+
+create_database_and_user() {
+    DB=$1
+    USER=$2
+    PASSWORD=$3
+    
+    mysql --user root --password=$MYSQL_ROOT_PASSWORD <<EOS
+CREATE DATABASE IF NOT EXISTS ${DB};
+CREATE USER IF NOT EXISTS  '${USER}'@'%' IDENTIFIED BY '${PASSWORD}';
+GRANT ALL ON ${DB}.* TO '${USER}'@'%';
+EOS
+}
+
+create_database_and_user icingaweb icingaweb ${ICINGAWEB_MYSQL_PASSWORD}
```

Comment:
Looks like defaults for the env vars are missing, i.e. `icingaweb` and `icingadb`.

Reply:
I think these are better set in the docker-compose file, so I set it there.